bpf/types.h

```diff
@@ -104,6 +104,9 @@ typedef struct flow_metrics_t {
     // https://chromium.googlesource.com/chromiumos/docs/+/master/constants/errnos.md
     u8 errno;
     u8 dscp;
+    u8 nb_observed_intf;
+    u8 observed_direction[MAX_OBSERVED_INTERFACES];
+    u32 observed_intf[MAX_OBSERVED_INTERFACES];
```

Comment:
why u changed this instead of having array of struct ?

Reply:
It's padding optimization. With a struct there was a 7 bytes padding per struct instance, so 42 bytes in total. Separated arrays reduce it to a minimum.